development/nautobot_config.py

```diff
@@ -58,6 +58,10 @@
                 "handlers": ["verbose_console" if DEBUG else "normal_console"],
                 "level": LOG_LEVEL,
             },
+            "": {
+                "handlers": ["verbose_console" if DEBUG else "normal_console"],
+                "level": LOG_LEVEL,
+            },
             "rq.worker": {
```

Comment:
Should be able to remove this one.

Reply:
Fixed here:
https://github.com/nautobot/nautobot-plugin-data-validation-engine/pull/37/files#diff-c21c8c3466cae2eb1a0aba98a64ed71d2d6cb7dd4b71661b5a8263471493e982L65